assets/less/makeup.less

```diff
@@ -17,6 +9,7 @@ code.makeup {
   }
 }
 
+@hljs-default-text-color: #4d4d4c;
```

Comment:
Can we call this just `@default-text-color`? One day we will remove hljs and this won't make any sense :D Thanks!

Reply:
@josevalim Done!

> One day we will remove hljs

This will be a great day, I think hljs causes some FOUCs